ui/app/AppLayouts/Communities/panels/MembersTabPanel.qml

```diff
@@ -70,23 +70,24 @@ Item {
 
                 sourceModel: root.model
 
-                proxyRoles: FastExpressionRole {
-                    function displayNameProxy(localNickname, ensName, displayName, aliasName) {
-                        return ProfileUtils.displayName(localNickname, ensName, displayName, aliasName);
-                    }
-
-                    name: "preferredDisplayName"
-                    expectedRoles: ["localNickname", "displayName", "ensName", "alias"]
-                    expression: displayNameProxy(model.localNickname, model.ensName, model.displayName, model.alias);
-                }
-
-
                 sorters : [
                     StringSorter {
                         roleName: "preferredDisplayName"
                         caseSensitivity: Qt.CaseInsensitive
                     }
                 ]
+
+                filters: ExpressionFilter {
+                    enabled: !!memberSearch.text
+
+                    expression: {
```

Comment:
Hmm, why not use the `preferredDisplayName`? I thought it's part of the model now